modules/text_server_fb/text_server_fb.h

```diff
@@ -116,6 +118,9 @@ class TextServerFallback : public TextServerExtension {
 	HashMap<StringName, int32_t> feature_sets;
 	HashMap<int32_t, StringName> feature_sets_inv;
 
+	std::atomic<TextServer::FontLCDSubpixelLayout> lcd_subpixel_layout = TextServer::FontLCDSubpixelLayout::FONT_LCD_SUBPIXEL_LAYOUT_NONE;
```

Comment:
Any reason this uses `std::atomic` and not `SafeNumeric`?

Reply:
i didn't know SafeNumeric existed, ive used std::atomic before and saw it was used in other places in godot

Reply:
I'd suggest checking it out it's in `core/templates/safe_refcount.h`

Reply:
should i change the `std::atomic` to `SafeNumeric`? it looks like a wrapper over `std::atomic` without the convenient operator overloads and conversion operators

Reply:
I'd say yes, raw `std::atomic` is used essentially only in the internal components utilizing them, with the small exception of being used in `RichTextLabel` and `AudioServer`, otherwise we use `SafeNumeric` everywhere, I'm not sure what is behind those exceptions so I'd say to use `SafeNumeric` unless there's some special reason not to, as it gives us a standardized interface (and more importantly, allows us to change the underlying implementation if we wish, to use platform specific stuff etc.)

Reply:
ah... on second thought i would have to change all of the reads of `valid` to use the member function `is_set()` for the exact same semantics .. i hope thats a good enough reason to not use `SafeNumeric`

Reply:
Personally I'd argue the opposite, that having explicit communication that this isn't a POD type but an atomic is a bonus, conveying that it should be given special attention especially for code that's already potentially working on these parts of the engine, and to avoid having to go to the header to see what type it actually is

Reply:
Maybe @RandomShaper can chime in as the resident atomic/SafeNumeric expert.

Reply:
makes sense